test/functional/wallet_send.py

```diff
@@ -184,6 +184,33 @@ def test_send(self, from_wallet, to_wallet=None, amount=None, data=None,
 
         return res
 
+    def test_maxfeerate(self):
```

Comment:
Maybe add tests where you (successfully) configure the node's wallet using `-maxfeerate`?

Reply:
Similarly, it would be good to test that both maxfee and maxfeerate requirements must be met.

Reply:
> Maybe add tests where you (successfully) configure the node's wallet using -maxfeerate?

I dont understand what you mean by successfully configure nodes wallet.

The test you are commenting on configure node[0] with the `-maxfeerate` option.
Could you clarify your comment, thanks.

---
>  Similarly, it would be good to test that both maxfee and maxfeerate requirements must be met.

Their is a test for `maxtxfee` already, and I added one for `maxfeerate`.
https://github.com/bitcoin/bitcoin/blob/11b436a66af3ceaebb0f907878715f331516a0bc/test/functional/wallet_bumpfee.py#L557